pkg/build/pipelines/ruby/install.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build and install a ruby gem
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
+
+inputs:
+  gem:
+    description: |
+      Gem name
+    required: true
+
+  version:
+    description: |
+      Gem version to install. This can be a version tag (v1.0.0)
+
+pipeline:
+  - runs: |
+      gem build ${{inputs.gem}}.gemspec
+      
+      gem install \
+      --local \
+      --install-dir ${{targets.destdir}}  \
+      --bindir ${{targets.destdir}}/usr/bin \
+      --ignore-dependencies \
+      --document ri \
+      --verbose \
+      -v ${{inputs.version}} \
```

Comment:
If `-v ${{inputs.version}}` is always used, then `version` should be `required`?

Reply:
good point